File: 백준/Bronze/2566. 최댓값/최댓값.cc
#include <iostream>
#include <string>
using namespace std;

int main(){
    int a[10][10], max = 0, n = 0, m = 0;
    for(int i = 0; i < 9; i++){
        for(int j = 0; j < 9; j++){
            cin >> a[i][j];
        }
    }
    for(int i = 0; i < 9; i++){
        for(int j = 0; j < 9; j++){
            if(a[i][j] > max){
                max = a[i][j];
                n = i;
                m = j;
            }
        }
    }
    cout << max << "\n" << n+1 << " " << m+1;
}

File: 백준/Bronze/2566. 최댓값/최댓값.swift
var tmp: [[Int]] = []

for _ in 0..<9 {
    let input = readLine()!.split(separator: " ").map{ Int($0)! }
    tmp.append(input)
}

let maxValue = tmp.flatMap{ $0 }.max()!

print(maxValue)
for i in 0..<9 {
    if tmp[i].max() == maxValue {
        print(i + 1, tmp[i].firstIndex(of: maxValue)! + 1)
        break
    }
}